src/main/java/com/inqoo/trainingservice/app/controller/SubcategoryController.java

```diff
@@ -0,0 +1,27 @@
+package com.inqoo.trainingservice.app.controller;
+
+import com.inqoo.trainingservice.app.models.Subcategory;
+import com.inqoo.trainingservice.app.service.SubcategoryService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/categories")
+public class SubcategoryController {
+    public SubcategoryController(SubcategoryService subcategoryService) {
+        this.subcategoryService = subcategoryService;
+    }
+
+    private SubcategoryService subcategoryService;
```

Comment:
Field declaration should be above constructor

Reply:
Done.